puma.gemspec

```diff
@@ -4,7 +4,7 @@ Gem::Specification.new do |s|
   s.name = "puma"
   s.version = Puma::Const::PUMA_VERSION
   s.authors = ["Evan Phoenix"]
-  s.description = "Puma is a simple, fast, threaded, and highly parallel HTTP 1.1 server for Ruby/Rack applications. Puma is intended for use in both development and production environments. It's great for highly parallel Ruby implementations such as Rubinius and JRuby as well as as providing process worker support to support CRuby well."
+  s.description = "Puma is a simple, fast, threaded, and highly parallel HTTP 1.1 server for Ruby/Rack applications. Puma is intended for use in both development and production environments. It's great for highly parallel Ruby implementations such as Rubinius and JRuby as well as providing process worker support to support CRuby well."
```

Comment:
While we're at it I think this outdated description should be updated. Rubinius is a not thing anymore. I think we can take the paragraph from the top of the README https://github.com/puma/puma for